app/src/main/java/io/github/gmathi/novellibrary/fragment/WebPageDBFragment.kt

```diff
@@ -198,31 +196,38 @@ class WebPageDBFragment : BaseFragment() {
 
     private fun loadData(liveFromWeb: Boolean = false) {
         doc = null
-        readerWebView.stopLoading()
-        readerWebView.clearView()
-        if (webPage?.filePath != null && !liveFromWeb)
+
+        readerWebView.apply {
+            stopLoading()
+            loadUrl("about:blank")
+        }
+
+        if (webPage?.filePath != null && !liveFromWeb) {
             loadFromFile()
-        else
+        } else {
             loadFromWeb()
+        }
```

Comment:
That auto clean & refactor... Those `if { } else { }` even for a single statement, need to clean that up later.

Reply:
Haha sorry. I just edited as I went along and I prefer using braces even for single statements in conditionals. I'll stop doing it.